### PR TITLE
Path: Fix #3914: Adding stock label prefix to name comparison in orde…

### PR DESCRIPTION
**Pull request by gauna85**
…r to select correct existing solid in combo box

Fixes  bug 3914 

https://tracker.freecadweb.org/view.php?id=3914

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

**Comment by gauna85** [comment]
Discussion:

https://forum.freecadweb.org/viewtopic.php?t=53226

File: src/Mod/Path/PathScripts/PathJobGui.py [hunk 1]
```diff
@@ -553,7 +554,9 @@ def setFields(self, obj):
         index = -1
         for i, solid in enumerate(self.candidates(obj)):
             self.form.stockExisting.addItem(solid.Label, solid)
-            if solid.Label == stockName:
+            label="%s-%s" % (self.StockLabelPrefix,solid.Label)
```
Comment (mlampert) [inline src/Mod/Path/PathScripts/PathJobGui.py:557]: FYI: we're trying to use the new formatting style

**Review by mlampert — APPROVED** [review]

File: src/Mod/Path/PathScripts/PathJobGui.py (hunk 1 above)
Comment (sliptonic) [inline src/Mod/Path/PathScripts/PathJobGui.py:557]: `label="{}-{}".format(self.StockLabelPrefix, solid.Label)`

Comment (sliptonic) [reply src/Mod/Path/PathScripts/PathJobGui.py:557]: I'm just playing with github.  I think there's a way to suggest a code change right in the web page and the author can accept it directly without having to do a roundtrip commit and push again.

Comment (chennes) [reply src/Mod/Path/PathScripts/PathJobGui.py:557]: I can't use markdown to demo markdown, but it's like
```
'''suggestion
            label="{}-{}".format(self.StockLabelPrefix, solid.Label)
'''
```

Comment (sliptonic) [reply src/Mod/Path/PathScripts/PathJobGui.py:557]: neat.  Thanks

**Comment by gauna85** [comment]
Nobody able to merge?

**Pull request merged**